.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: false
```

Comment:
wont this apply to org members

Reply:
yeah

Reply:
then we can't make issues for version projects

Reply:
wdym

Reply:
thinking about it better, ig it doesnt hurt to have it enabled, if we see its a bad idea we can just disable it later